autopush/websocket.py

```diff
@@ -304,6 +309,8 @@ def onMessage(self, payload, isBinary):
             return self.process_ack(data)
         else:
             self.sendClose()
+        # Done processing, start idle.
+        self.set_active()
```

Comment:
What do you think about moving `self.check_idle()` below this line, so that you don't need to call it in `process_notifications`, `process_ping`, etc.?